tests/mir-opt/lower_intrinsics.rs

```diff
@@ -267,3 +267,20 @@ pub fn get_metadata(a: *const i32, b: *const [u8], c: *const dyn std::fmt::Debug
     let _usize = ptr_metadata(b);
     let _vtable = ptr_metadata(c);
 }
+
+// Check that the MIR shims used for reifying intrinsics to `fn` pointers,
+// also go through the lowering pass.
+pub fn reify_intrinsics() -> impl Copy {
```

Comment:
Question: are u maybe missing a `// EMIT_MIR` mir-opt tool instruction here? See fns above this.